test_template.yml

```diff
@@ -88,10 +107,29 @@ jobs:
         with:
           packages: xdot rolldice
           version: ${{ github.run_id }}-${{ github.run_attempt }}
+      - id: check-additional-pkg-dir
```

Comment:
Usually prefer to use name instead of id. They are functionally the same in the way it is being used but would be good to make it easier to read.